ansible/hosts.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
nit: I think this would be better as a git mv

Reply:
noted, but im not sure how to do this cleanly now.  will do it in the future.